node/node_newblock.go

```diff
@@ -31,7 +31,7 @@ func (node *Node) WaitForConsensusReady(readySignal chan struct{}, stopChan chan
 			// keep waiting for Consensus ready
 			select {
 			case <-readySignal:
-				time.Sleep(100 * time.Millisecond) // Delay a bit so validator is catched up (test-only).
+				time.Sleep(5000 * time.Millisecond) // Delay a bit so validator is catched up (test-only).
```

Comment:
5 seconds? why?

Reply:
This PR is not ready for review yet. I will clean up after it works with the master code.

Reply:
reverted